src/constants.js

```diff
@@ -22,3 +22,9 @@
 
 // eslint-disable-next-line import/prefer-default-export
 
+export const AW_SUB_URL = {
+  ORG_GROUP: 'system/groups/',
+  ORG_GROUP_CHILDREN: '/children',
+};
+
+export const AW_ROOT_ORG_GROUP_ID = 574;
```

Comment:
I wouldn't put this in as the default group because its specific to our usage. It might be different for anyone else who wants to use the package. It should be a constant in the API and/or Agent.